core/src/main/java/com/datastrato/gravitino/utils/EntitySpecificConstants.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.utils;
+
+public class EntitySpecificConstants {
+
+  private EntitySpecificConstants() {}
+
+  public static final String SYSTEM_METALAKE_RESERVED_NAME = "system";
+  public static final String SYSTEM_CATALOG_RESERVED_NAME = "system";
+  public static final String AUTHORIZATION_CATALOG_NAME = "authorization";
+  public static final String USER_SCHEMA_NAME = "user";
+  public static final String GROUP_SCHEMA_NAME = "group";
+  public static final String ADMIN_SCHEMA_NAME = "admin";
+}
```

Comment:
I don't think we have to create a new file for this and put it in `utils`, we can put all of them in `Entity`, WDYT?

Reply:
This is not fixed, right?

Reply:
This is fixed. I have moved it to core/src/main/java/com/datastrato/gravitino/meta/EntitySpecificConstants.java

Reply:
> we can put all of them in Entity

I suggest we move all of them to `Entity` class.

Reply:
I think you still don't address this issue like what I mentioned above, right?

Reply:
Sorry to miss this comment. I will correct it.

Reply:
Done.